engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/MethodDefinitionExtractor.scala

```diff
@@ -2,16 +2,19 @@ package pl.touk.nussknacker.engine.definition
 
 import java.lang.annotation.Annotation
 import java.lang.reflect.Method
-
+import com.github.ghik.silencer.silent
```

Comment:
we don't use silent and nowarn here, i haven't  found usage of deprecated methods here, probably can be removed

Reply:
:+1: